### ISSUE TITLE
Empty plots in compare_cumulative_psf.py

### ISSUE THREAD

**Issue by GernotMaier** [gammasim/gammasim-tools#181]
While trying the example

```
 python gammasim-tools/applications/compare_cumulative_psf.py --site North --telescope LST-1 --model_version prod4 --pars lst_pars.yml --data PSFcurve_data_v2.txt
```
I get a good looking cumulative plot (see attached), but an empty image (also attached).

I tried to solve it, but failed. `dataToPlot` in [line 203](https://github.com/gammasim/gammasim-tools/blob/251126c707f65d56916850d51ba56030518ec8da/applications/compare_cumulative_psf.py#L203) is filled correctly, but somehow the 2D plotting fails.

![compare_cumulative_psf_LST-1_cumulativePSF](https://user-images.githubusercontent.com/14001688/152792346-a0490b9c-4931-4098-bd18-08dae91065a8.png)
![compare_cumulative_psf_LST-1_image](https://user-images.githubusercontent.com/14001688/152792418-f2b44ad2-5a39-4457-9aa2-823914b7e0e9.png)

**Comment by RaulRPrado** [comment]
Fixed in #182

**Comment by GernotMaier** [comment]
Works - thank you!
